packages/js/src/preferences/preference.ts

```diff
@@ -0,0 +1,38 @@
+import { ApiService } from '@novu/client';
+
+import { NovuEventEmitter } from '../event-emitter';
+import { ChannelPreference, ChannelPreferenceOverride, ChannelType, PreferenceLevel, WorkflowInfo } from '../types';
+import { ApiServiceSingleton } from '../utils/api-service-signleton';
+import { updatePreference } from './helpers';
```

Comment:
Nit-picky suggestion: Move the helpers into the Preference class to have everything in one place and and avoid an extra import. Every byte counts ;)

Reply:
can't do that as it's reused in two places: single Preference class and Preferences module